tests/providers/google/cloud/operators/test_dataprep_system.py

```diff
@@ -0,0 +1,45 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import json
+from os import environ
+
+from airflow.models import Connection
+from airflow.utils.session import create_session
+from tests.test_utils.db import clear_db_connections
+from tests.test_utils.gcp_system_helpers import CLOUD_DAG_FOLDER, GoogleSystemTest
+
+TOKEN = environ.get("DATAPREP_TOKEN", "dataprep-system-test-token")
+EXTRA = {"token": TOKEN}
+
+
+class DataprepExampleDagsTest(GoogleSystemTest):
```

Comment:
We don't use Google authorization, so we shouldn't use GoogleSystemTest.

Reply:
But it's in google providers package so we should be able to run it using `--system=google`

Reply:
Good point, but we should still detect missing credentials.

Reply:
I do not understand what is the problem here.

Reply:
The test will fail if there's no real token. Developers should be aware of that. You should check if the system variable is set. If not then skip the test with useful message

Reply:
I talked about it with Kamil yesterday and he explained me everything.